cmd/install_probe.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(installProbeCmd)
+}
+
+var installProbeCmd = &cobra.Command{
+	Use:   "install-probe",
+	Short: "Join the community powered Globalping platform by running a Docker container.",
+	Long: `Pull and run the Globalping probe Docker container on this machine. It requires Docker to be installed.`,
+	Run: installProbeCmdRun,
+}
+
+func installProbeCmdRun(cmd *cobra.Command, args []string) {
+	dockerInfoCmd := exec.Command("docker", "info")
+	dockerInfoCmd.Stderr = os.Stderr
+	err := dockerInfoCmd.Run()
+	if err != nil {
+		fmt.Printf("docker info command failed: %v\n\n", err)
+		fmt.Println("Docker was not detected on your system and it is required to run the Globalping probe. Please install Docker and try again.")
+		return
+	}
+
+	dockerInspectCmd := exec.Command("docker", "inspect", "globalping-probe", "-f", "{{.State.Status}}")
+	containerStatus, err := dockerInspectCmd.Output()
+	if err == nil {
+		containerStatusStr := string(bytes.TrimSpace(containerStatus))
+		fmt.Printf("The globalping-probe container is already installed on your system. Current status: %s\n", containerStatusStr)
+		return
+	}
+
+	ok := askUser("The Globalping platform is a community powered project and relies on individuals like yourself to host our probes and make them accessible to everyone else. Please confirm to pull and run our Docker container (ghcr.io/jsdelivr/globalping-probe) ")
```

Comment:
Can you add a new line after "everyone else." ? It will look better in the terminal